webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/NodeApi.scala

```diff
@@ -584,8 +583,8 @@ class NodeApiService13 (
     , properties            : List[NodeProperty]
     , inheritedProperties   : List[NodePropertyHierarchy]
     , softs                 : List[Software]
-    , compliance            : Option[NodeStatusReport]
-    , sysCompliance         : Option[NodeStatusReport]
+    , compliance            : Option[ComplianceLevel]
+    , sysCompliance         : Option[ComplianceLevel]
```

Comment:
this changes make the serialization nearly twice as fast : ~ 400ms vs 700ms

Reply:
wow //  cc @VinceMacBuche for info

Reply:
it because the serialization used to compute the compliance, which is costly. now it is done before, see https://github.com/Normation/rudder/pull/4018#discussion_r758324027